src/framework/midi/internal/platform/osx/coremidiinport.cpp

```diff
@@ -187,6 +187,7 @@ void CoreMidiInPort::initCore()
     }
 
     QString portName = "MuseScore MIDI input port";
+    static bool doLog = false; // kors::logger::Logger::instance()->isLevel(kors::logger::Level::Debug);
```

Comment:
See `audiobuffer.cpp` for the preferred way for such things; for example at the top of the file
```cpp
//#define DEBUG_COREMIDIINPORT
#ifdef DEBUG_COREMIDIINPORT
#define LOG_MIDI LOGD
#else
#define LOG_MIDI LOGN
#endif
```
And then just write
```cpp
                LOG_MIDI() << "Receiving MIDIEventPacket with " << packet->wordCount << " words";
```

Reply:
I have changed this.